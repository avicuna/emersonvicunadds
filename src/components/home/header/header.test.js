import React from 'react';
import Header from './header';
import {shallow} from 'enzyme';

describe('Header', () => {
    test('UI renders correctly', () => {
        const wrapper = shallow(<Header/>);
        expect(wrapper).toMatchSnapshot();
    });
});