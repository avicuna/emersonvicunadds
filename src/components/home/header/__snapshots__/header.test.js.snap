// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header UI renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <header>
        <t
          tag="div"
        >
          <img
            alt="stuff"
            className="img-fluid abq"
            src="albuquerque-skyline.jpg"
          />
          <t
            className="flex-center  text-white text-center"
            overlay=""
            style={
              Object {
                "flexDirection": "column",
              }
            }
            tag="div"
          >
            <h1
              className="display-3"
            >
              Emerson Vicuña, DDS Family Dentistry
            </h1>
            <h2>
              Albuquerque, New Mexico
            </h2>
          </t>
        </t>
      </header>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <t
          tag="div"
        >
          <img
            alt="stuff"
            className="img-fluid abq"
            src="albuquerque-skyline.jpg"
          />
          <t
            className="flex-center  text-white text-center"
            overlay=""
            style={
              Object {
                "flexDirection": "column",
              }
            }
            tag="div"
          >
            <h1
              className="display-3"
            >
              Emerson Vicuña, DDS Family Dentistry
            </h1>
            <h2>
              Albuquerque, New Mexico
            </h2>
          </t>
        </t>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <img
              alt="stuff"
              className="img-fluid abq"
              src="albuquerque-skyline.jpg"
            />,
            <t
              className="flex-center  text-white text-center"
              overlay=""
              style={
                Object {
                  "flexDirection": "column",
                }
              }
              tag="div"
            >
              <h1
                className="display-3"
              >
                Emerson Vicuña, DDS Family Dentistry
              </h1>
              <h2>
                Albuquerque, New Mexico
              </h2>
            </t>,
          ],
          "tag": "div",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "stuff",
              "className": "img-fluid abq",
              "src": "albuquerque-skyline.jpg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h1
                  className="display-3"
                >
                  Emerson Vicuña, DDS Family Dentistry
                </h1>,
                <h2>
                  Albuquerque, New Mexico
                </h2>,
              ],
              "className": "flex-center  text-white text-center",
              "overlay": "",
              "style": Object {
                "flexDirection": "column",
              },
              "tag": "div",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Emerson Vicuña, DDS Family Dentistry",
                  "className": "display-3",
                },
                "ref": null,
                "rendered": "Emerson Vicuña, DDS Family Dentistry",
                "type": "h1",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Albuquerque, New Mexico",
                },
                "ref": null,
                "rendered": "Albuquerque, New Mexico",
                "type": "h2",
              },
            ],
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "header",
    },
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <header>
          <t
            tag="div"
          >
            <img
              alt="stuff"
              className="img-fluid abq"
              src="albuquerque-skyline.jpg"
            />
            <t
              className="flex-center  text-white text-center"
              overlay=""
              style={
                Object {
                  "flexDirection": "column",
                }
              }
              tag="div"
            >
              <h1
                className="display-3"
              >
                Emerson Vicuña, DDS Family Dentistry
              </h1>
              <h2>
                Albuquerque, New Mexico
              </h2>
            </t>
          </t>
        </header>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <t
            tag="div"
          >
            <img
              alt="stuff"
              className="img-fluid abq"
              src="albuquerque-skyline.jpg"
            />
            <t
              className="flex-center  text-white text-center"
              overlay=""
              style={
                Object {
                  "flexDirection": "column",
                }
              }
              tag="div"
            >
              <h1
                className="display-3"
              >
                Emerson Vicuña, DDS Family Dentistry
              </h1>
              <h2>
                Albuquerque, New Mexico
              </h2>
            </t>
          </t>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <img
                alt="stuff"
                className="img-fluid abq"
                src="albuquerque-skyline.jpg"
              />,
              <t
                className="flex-center  text-white text-center"
                overlay=""
                style={
                  Object {
                    "flexDirection": "column",
                  }
                }
                tag="div"
              >
                <h1
                  className="display-3"
                >
                  Emerson Vicuña, DDS Family Dentistry
                </h1>
                <h2>
                  Albuquerque, New Mexico
                </h2>
              </t>,
            ],
            "tag": "div",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "stuff",
                "className": "img-fluid abq",
                "src": "albuquerque-skyline.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h1
                    className="display-3"
                  >
                    Emerson Vicuña, DDS Family Dentistry
                  </h1>,
                  <h2>
                    Albuquerque, New Mexico
                  </h2>,
                ],
                "className": "flex-center  text-white text-center",
                "overlay": "",
                "style": Object {
                  "flexDirection": "column",
                },
                "tag": "div",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Emerson Vicuña, DDS Family Dentistry",
                    "className": "display-3",
                  },
                  "ref": null,
                  "rendered": "Emerson Vicuña, DDS Family Dentistry",
                  "type": "h1",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Albuquerque, New Mexico",
                  },
                  "ref": null,
                  "rendered": "Albuquerque, New Mexico",
                  "type": "h2",
                },
              ],
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": "header",
      },
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
